src/desktop/components/util/test/activator.test.coffee

```diff
@@ -25,7 +23,9 @@ describe 'Activator', ->
         @$cases = $(render('activator')())
         done()
 
-    after -> @$cases.remove()
+    after ->
+      benv.teardown()
```

Comment:
Could this be hoisted to an `afterEach`?

Reply:
Potentially, but wanted to match the `before` used here.